ui/node/identifiers.go

```diff
@@ -27,4 +27,12 @@ const (
 	WebAuthnRegisterDisplayName = "webauthn_register_displayname"
 	WebAuthnRemove              = "webauthn_remove"
 	WebAuthnScript              = "webauthn_script"
+	PasskeyRegisterTrigger      = "passkey_register_trigger"
+	PasskeyRegister             = "passkey_register"
+	PasskeySettingsRegister     = "passkey_settings_register"
```

Comment:
What is `settingsRegister`'s responsibility?

Reply:
The corresponding input field holds the browser response to the passkey challenge.

See: https://github.com/ory/kratos/blob/60a1817c6660d1eb668a9994026da385416dcf44/x/webauthnx/js/webauthn.js#L316